smriprep/workflows/outputs.py

```diff
@@ -239,25 +235,25 @@ def init_anat_derivatives_wf(bids_root, freesurfer, num_t1w, output_dir,
     if not freesurfer:
         return workflow
 
+    from niworkflows.interfaces.surf import Path2BIDS
+
     # FS native space transforms
     lta2itk_fwd = pe.Node(LTAConvert(out_itk=True), name='lta2itk_fwd')
     lta2itk_inv = pe.Node(LTAConvert(out_itk=True), name='lta2itk_inv')
     ds_t1w_fsnative = pe.Node(
-        DerivativesDataSink(base_directory=output_dir, allowed_entities=['from', 'to', 'mode'],
+        DerivativesDataSink(base_directory=output_dir,
                             mode='image', to='fsnative', suffix='xfm', **{'from': 'T1w'}),
         name='ds_t1w_fsnative', run_without_submitting=True)
     ds_fsnative_t1w = pe.Node(
-        DerivativesDataSink(base_directory=output_dir, allowed_entities=['from', 'to', 'mode'],
+        DerivativesDataSink(base_directory=output_dir,
                             mode='image', to='T1w', suffix='xfm', **{'from': 'fsnative'}),
         name='ds_fsnative_t1w', run_without_submitting=True)
     # Surfaces
-    name_surfs = pe.MapNode(GiftiNameSource(
```

Comment:
After this, we can deprecate `GiftiNameSource` if we think we won't be using it anymore.